pennylane/ops/__init__.py

```diff
@@ -34,6 +34,7 @@
 from .qubit import obs as _qubit__obs__
 
 
+
```

Comment:
Why this extra line space? :thinking:

Reply:
Yeah, I made changes to this file but decided to add them in a later PR.